doc/Developing/Using-Git.md

```diff
@@ -39,7 +39,7 @@ git clone git@github.com:/yourusername/librenms.git
 ```
 
 If you are starting with an existing LibreNMS install either from our Virtual machine or install docs then you 
-will need to update the remote url. To do that run:
+will need to update the remote url. Be aware that daily.sh will not work properly after changing your origin. To do that run:
```

Comment:
What do you see happening here?

It may just need `git branch --set-upstream-to=origin/master master` setting?